docs/_themes/sylius_rtd_theme/breadcrumbs.html

```diff
@@ -14,8 +14,13 @@
       <li class="navbar__separator">|</li>
       <li class="wy-breadcrumbs-aside">
         {% if display_github %}
-        <a href="https://github.com/{{ github_user }}/{{ github_repo }}/blob/{{ github_version }}{{ conf_py_path }}{{ pagename }}{{ source_suffix }}"
-          class="fa fa-github"> Edit on GitHub</a>
+          {% if sourcename == null %}
+            <a href="https://github.com/{{ github_user }}/{{ github_repo }}/blob/docs"
```

Comment:
We should redirect to this link always, I guess:
`https://github.com/Sylius/Sylius/tree/master/docs`

Unfortunately, it doesn't work right :<